stablehlo/conversions/linalg/transforms/StablehloLegalizeToLinalg.cpp

```diff
@@ -1899,10 +1899,8 @@ struct SelectAndScatterNoOverlapConverter final
     if (!op.getWindowDimensions().has_value())
       return rewriter.notifyMatchFailure(op, "no window dimensions found");
 
-    auto strides =
-        llvm::to_vector(op.getWindowStridesAttr().getValues<int64_t>());
-    auto window =
-        llvm::to_vector(op.getWindowDimensionsAttr().getValues<int64_t>());
+    auto strides = op.getWindowStrides().value_or(SmallVector<int64_t>{});
```

Comment:
The two if-stmts above check that there is a value, can we just use `value()` here instead of `value_or` ?